lib/hex_web/store/s3.ex

```diff
@@ -1,36 +1,24 @@
 defmodule HexWeb.Store.S3 do
   @behaviour HexWeb.Store
-  @max_attempts 10
-
-  defmacrop s3_config(opts) do
-    quote do
-      {:config,
-        unquote(opts[:url]),
-        unquote(opts[:access_key_id]),
-        unquote(opts[:secret_access_key]),
-        :virtual_hosted}
-    end
-  end
+  alias ExAws.S3
 
   def list_logs(prefix) do
     list(:logs_bucket, prefix)
   end
 
   def get_logs(key) do
-    bucket = Application.get_env(:hex_web, :logs_bucket) |> String.to_char_list
-    key = String.to_char_list(key)
-    result = retry(fn -> :mini_s3.get_object(bucket, key, [], config()) end)
-    result[:content]
+    Application.get_env(:hex_web, :logs_bucket)
+    |> S3.get_object!(key)
+    |> Map.get(:body)
```

Comment:
Wondering wether this should be `Dict.fetch!(:body)` in the spirit of raising when missing...